store/store.go

```diff
@@ -549,10 +550,23 @@ func (s *Store) RemoveACI(key string) error {
 		}
 
 		uid := os.Getuid()
-		dirUid := int(fi.Sys().(*syscall.Stat_t).Uid)
+		dirGid := int(fi.Sys().(*syscall.Stat_t).Gid)
 
-		if uid != dirUid && uid != 0 {
-			return fmt.Errorf("permission denied on %s, directory owner id (%d) does not match current real id (%d). Are you root or the owner of the image?", path, dirUid, uid)
+		gids, err := os.Getgroups()
+		if err != nil {
+			return errwrap.Wrap(errors.New("error getting our own groups"), err)
+		}
+		gids = append(gids, os.Getgid())
+
+		userInDirGroup := false
+		for _, g := range gids {
+			if g == dirGid {
+				userInDirGroup = true
+			}
+		}
+
+		if !userInDirGroup && uid != 0 {
+			return fmt.Errorf("permission denied on %s, user not in the directory group id (%d). Are you root or in the %q group?", path, dirGid, common.RktGroup)
 		}
```

Comment:
Right, it's just better UX.

I guess there're two possibilities:

* You are in the rkt group or root: Everything is fine
* You are not in the rkt group: You'll get errors before this function.

I guess we can remove it then. It was there before because it was depending on the user that fetched the image but now all users on the rkt group can do anything with images.